cadvisor.go

```diff
@@ -61,6 +61,7 @@ var collectorCert = flag.String("collector_cert", "", "Collector's certificate,
 var collectorKey = flag.String("collector_key", "", "Key for the collector's certificate")
 
 var storeContainerLabels = flag.Bool("store_container_labels", true, "convert container labels and environment variables into labels on prometheus metrics for each container. If flag set to false, then only metrics exported are container name, first alias, and image name")
+var whiteListedContainerLabels = flag.String("white_listed_container_labels", "", "comma separated list of container labels to be converted to labels on prometheus metrics for each container.")
```

Comment:
I think we want to explicitly say that you need to specify `store_container_labels=false` to use the whitelistedContainerLables.

Reply:
whitelisted is one word, not two (even though my browser disagrees).  The flag should be `whitelisted_container_labels`, and variables should be `whitelistedFoo`.